domain.go

```diff
@@ -312,6 +312,7 @@ type DomainGraphicListener struct {
 }
 
 type DomainGraphic struct {
+	XMLName       xml.Name                `xml:"graphics"`
```

Comment:
Why do you think this is needed ? It is only something that needs to be added for structs which form the root of an XML document - DomainGraphic is only ever needed as a child of another element.